lib/web_ui/lib/src/engine/dom_renderer.dart

```diff
@@ -161,8 +161,8 @@ class DomRenderer {
   html.Element? _glassPaneElement;
 
   /// The ShadowRoot of the [glassPaneElement].
```

Comment:
Since it's no longer necessarily a shadow root, we should update the doc comment. We could say something like "For example, in browsers that support shadow DOM this is the shadow root." though.

Reply:
I'm going to search for ShadowRoot mentions and update the comments, good point.